web_src/css/repo/issue-list.css

```diff
@@ -1,35 +1,64 @@
-.issue-list-toolbar {
-  display: flex;
-  flex-wrap: wrap;
-  align-items: flex-start;
-  gap: 1rem;
-  margin-top: 1rem;
-}
+@media only screen and (min-width: 1200px) {
+  #issue-filters {
+    margin-top: 0;
+    margin-left: auto;
+  }
 
-.issue-list-toolbar-left {
-  display: flex;
-}
+  #issue-filters > .issue-list-toolbar-right > .issues-menu {
+    justify-content: flex-end;
+  }
 
-.issue-list-toolbar-right {
-  margin-left: auto;
+  #issue-filters > .issue-list-toolbar-left {
+    padding-left: 0;
+  }
 }
 
-@media (max-width: 767.98px) {
-  .issue-list-toolbar {
-    flex-direction: column-reverse;
-  }
-  .issue-list-toolbar-right {
+@media only screen and  (max-width: 769px) {
+  #issue-filters > .issue-list-toolbar-right {
     margin-right: auto;
-    width: 100%;
+    width: 100% !important;
+    padding-top: 0;
+
+  }
+
+  #issue-filters > .issue-list-toolbar-right > .issues-menu {
+    flex-direction: row;
+    overflow-x: scroll;
+  }
+
+  #issue-filters > .issue-list-toolbar-right > .issues-menu > .item {
+    max-width: fit-content;
   }
-  .issue-list-navbar {
-    order: 0;
+
+  #issue-filters > .issue-list-toolbar-left {
+    margin-top: 10px !important;
+  }
+
+  .ui.stackable.menu:not(.no-vertical-tabs) > .dropdown.item {
```

Comment:
What is `no-vertical-tabs`? I can't find it in code or in Fomantic (correct me if I am wrong)

Reply:
Changed, I took it from chrome[Dev tools styles selector]